src/exchange/calculate.go

```diff
@@ -12,6 +12,8 @@ import (
 )
 
 const (
+	// SatoshiExponent is the number of decimal places for 1 satoshi
+	SatoshiExponent int64 = 8
 	// SatoshisPerBTC is the number of satoshis per 1 BTC
 	SatoshisPerBTC int64 = 1e8
```

Comment:
Why the additional constant? Is the old constant still necessary?

Reply:
converting from BTC to Satoshi and vice versa 
As internally we store deposits in form of Satoshi but c2cx requires in BTC